website/pages/docs/drivers/external/iis.mdx

```diff
@@ -0,0 +1,104 @@
+---
+layout: docs
+page_title: 'Drivers: nomad-driver-iis'
+sidebar_title: Windows IIS
+description: >-
+  The IIS driver is used for running
+  Windows IIS services.
+---
+
+# Windows IIS Driver
+
+Name: `nomad-driver-iis`
+
+The nomad iis driver provides an interface for running windows iis website tasks.
+A "Website" is a combination of an application pool and a site (app, vdir, etc.).
+Each allocation will create an application pool and site with the name being the allocation ID (guid).
+
+For more detailed instructions on how to set up and use this driver, please
+refer to the [documentation][nomad-driver-iis].
```

Comment:
You can take or leave this one. I just think it might be confusing to "refer to the documentation" when this is already technically documentation. Just different docs. 🤷 

```suggestion
refer to the [project documentation][nomad-driver-iis].
```

Reply:
Fixed. Changed it to `project README.md`.